android/app/src/main/res/values/styles.xml

```diff
@@ -72,7 +72,7 @@
 
     <style name="BootTheme" parent="Theme.SplashScreen">
         <item name="android:statusBarColor">@android:color/transparent</item>
-        <item name="windowSplashScreenAnimatedIcon">@mipmap/bootsplash_logo</item>
+        <item name="windowSplashScreenAnimatedIcon">@drawable/bootsplash_logo</item>
```

Comment:
I moved the bootsplash logo from `mipmap-*` to `drawable-*` directories to align with [`react-native-bootsplash` v5](https://github.com/zoontek/react-native-bootsplash/releases/tag/5.0.0) and [AndroidX `core-splashscreen`](https://developer.android.com/develop/ui/views/launch/splash-screen/migrate)